### PR TITLE
Improve LCG section of /install

### PR DESCRIPTION
**Pull request by eguiraud**
Slightly improved wording, examples updated to v6.24.

### CODE REVIEW

**Comment by eguiraud** [comment]
@Axel-Naumann is this good to go?

File: install/index.md [hunk 1]
```diff
@@ -163,38 +163,42 @@ The same instructions as [for Linux](#nixos_nix_nixpkgs) apply when running on m
...
-For example, to set up ROOT 6.22/00 on a CentOS7 machine that already has all [ROOT required dependencies]({{'/install/dependencies' | relative_url}}) installed, just run:
+For example, to set up ROOT 6.24/00 on a CentOS7 machine, just run:
 
 ```
-source /cvmfs/sft.cern.ch/lcg/app/releases/ROOT/6.22.00/x86_64-centos7-gcc48-opt/bin/thisroot.sh
+source /cvmfs/sft.cern.ch/lcg/app/releases/ROOT/6.24.00/x86_64-centos7-gcc48-opt/bin/thisroot.sh
 ```
 
-## ROOT plus dependencies
+Make sure to use the same compiler in use by the release in your working environment.
```
Comment (Axel-Naumann) [inline install/index.md:183]: I don't understand what this warning is trying to warn about. Maybe "Make sure you use the distribution's default compiler, just like this ROOT build"?

**Review by Axel-Naumann — APPROVED** [review]
LGTM (modulo possibly clarifying that warning)

File: install/index.md (hunk 1 above)
Comment (eguiraud) [reply install/index.md:183]: yes that's what i wanted to warn about, will rephrase

Comment (Axel-Naumann) [reply install/index.md:183]: Just merge when done, please

**Pull request merged**